utilities/boostHistHelpers.py

```diff
@@ -13,9 +13,27 @@ def valsAndVariances(h1, h2):
 def broadcastSystHist(h1, h2):
     if h1.ndim > h2.ndim or h1.shape == h2.shape:
         return h1
-    
+
+    a1 = h1.view(flow=True)
+    a2 = h2.view(flow=True)
+
+    # the additional axes have to be trailing, if they ar not, they must be moved
+    moves = []
+    offset = 0
+    for i, a in enumerate(h1.axes.name):
+        b = h2.axes.name[i+offset]
```

Comment:
In principle you could relax the assumption that the axes maintain the same order apart from an offset. Would just need to check that the axis exists and then find its index

Reply:
Not sure if I understand this point. I made some modifications here because I noticed we don't need to move the axes but can work only with the list of dimensions